app/load_dictionaries.py

```diff
@@ -0,0 +1,18 @@
+import csv
+import os
+
+def load_dict(file):
```

Comment:
You may have committed this prior to our conversations earlier today, but do you want to remove this function/file since the dictionaries will be load into the Word Class rather than the dictionary?

Reply:
Yeah this function, and likely the entire file, can be removed now, though I will need to rebase this branch onto development and do a bit of integration once the PRs with the word class